types/stylelint/index.d.ts

```diff
@@ -568,6 +570,11 @@ declare namespace stylelint {
 			},
 			RejectedMessage<[property: string]>
 		>;
+		'rule-nesting-at-rule-required-list': CoreRule<
+			OneOrMany<StringOrRegex>,
+			{},
+			ExpectedMessage<[patterns: string[]]>
```

Comment:
I forgot one case, if one use a js configuration you can pass `RegExp` instances directly.
Hence I think it should be
```suggestion
			ExpectedMessage<[patterns: StringOrRegex[]]>
```
right?